### PR TITLE
Ethereum tx data

### LINKED ISSUE #1854: ETH-like transaction with data doesn't show data anywhere
Maybe it should be in transaction detail?
all-trop account1
https://ropsten1.trezor.io/tx/0x0ec1f4eb8479bfb345f4b05406deb5c9a77691627b99af488498156678eddf6f

`"input": "0xdead",` field in detail (except for token transfers where input is a smart contract script)

[Design](https://www.figma.com/design/pIibgxmxD0jrAa9UGJd9da/Neue-Desktop-1.5?t=i6xNamdoKAz2amna-0)

### PR DESCRIPTION
**Pull request by tomasklim**
## Related Issue

Resolve #1854 oldie

## Screenshots:

Screenshot 2024-10-09 at 18 53 53

### CODE REVIEW

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/Data.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
+import { WalletAccountTransaction } from '@suite-common/wallet-types';
+import { Paragraph, Row } from '@trezor/components';
+import { palette, spacingsPx } from '@trezor/theme';
+import styled from 'styled-components';
+import { Translation } from 'src/components/suite';
+import { TranslationKey } from '@suite-common/intl-types';
+
+const Container = styled.div`
```
Comment (jvaclavik) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/Data.tsx:8]: You might use `Grid` component there.

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/Data.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    display: grid;
+    grid-template-columns: 130px 3fr;
+    gap: ${spacingsPx.md};
+    margin-top: ${spacingsPx.xxl};
+`;
+
+const ParagraphWrapper = styled.div`
+    white-space: pre-wrap;
+    overflow-wrap: anywhere;
```
Comment (jvaclavik) [inline packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/Data.tsx:17]: @adamhavel we might consider adding these to `textProps`. WDYT?

**Review by jvaclavik — APPROVED** [review]

File: packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/Data.tsx (hunk 1 above)
Comment (tomasklim) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/Data.tsx:8]: how do I do 
```
    grid-template-columns: 130px 3fr;
```

There is this in the `Grid`
```
    grid-template-columns: repeat(${({ $columns }) => $columns}, 1fr);
```

Comment (jvaclavik) [reply packages/suite/src/components/suite/modals/ReduxModal/UserContextModal/TxDetailModal/AdvancedTxDetails/Data.tsx:8]: So maybe not 😂 let's skip it

**Pull request merged**